dataframe/dataframe.go

```diff
@@ -52,9 +53,13 @@ var (
 )
 
 func readCsv(thread *starlark.Thread, _ *starlark.Builtin, args starlark.Tuple, kwargs []starlark.Tuple) (starlark.Value, error) {
+	return starlark.None, fmt.Errorf("dataframe.read_csv is disabled, use dataframe.parse_csv(string) instead to parse csv text that was already downloaded using the http package. In the future, dataframe.read_csv(url) will be restored")
+}
+
+func parseCsv(thread *starlark.Thread, _ *starlark.Builtin, args starlark.Tuple, kwargs []starlark.Tuple) (starlark.Value, error) {
 	var content starlark.Value
 
-	if err := starlark.UnpackArgs("read_csv", args, kwargs,
+	if err := starlark.UnpackArgs("parse_csv", args, kwargs,
```

Comment:
hrm, I'm not a huge fan of any precedent that has us breaking from the pandas API, but I agree this case it makes sense. Let's get a conversation going around a streaming interface in starlark, and use this in the meantime.

Reply:
This is actually un-breaking from the pandas API, since the original read_csv function did not match pandas's. We'll eventually reach parity, I think, but the `http` and `download` question (aka static analysis) needs to be in place first.